pkg/v14/resource/kubeconfig/desired_test.go

```diff
@@ -165,3 +166,11 @@ func Test_Resource_GetDesiredState(t *testing.T) {
 		})
 	}
 }
+
+func toCR(obj interface{}) (v1alpha1.ClusterGuestConfig, error) {
```

Comment:
```suggestion
func toClusterConfigCR(obj interface{}) (v1alpha1.ClusterGuestConfig, error) {
```

This naming is clearer for me.